cypress/integration/smoke/BasicTest.ts

```diff
@@ -39,4 +39,21 @@ describe("Basic Elements", () => {
       .find("a[data-testid='tableInstances-linkInstance']")
       .should("be.visible");
   });
+
+  it("Instance header details are available", () => {
+    const instanceHeaderDetails = [
+      "Name",
+      "Description",
+      "Status",
+      "Time created",
+    ];
+    cy.get(
```

Comment:
You can use the `ouiaId` command. See https://github.com/sjamboro/sandbox-ui/blob/MGDOBR-654/cypress/integration/smoke/BasicTest.ts#L33-L39